docs/tutorials/proposals/asset-transfer-proposal.md

```diff
@@ -105,6 +105,7 @@ You will need to define the dispatch strategy, which includes the metric, the le
   "distributionStrategy": "DISTRIBUTION_STRATEGY_PRO_RATA"
 ```
 
+## Submitting proposals in a batch
```

Comment:
The JSON snippets above are invalid JSON. I'm not sure if this matters, but to make them valid:
1. On the first line of each, the whole thing should be an object (an array is also valid, but not relevant here)
2. Close the `dispatchStrategy` and the object created in step 1.

```json
{ 
 "dispatchStrategy": {
  "assetForMetric": "b340c130096819428a62e5df407fd6abe66e444b89ad64f670beb98621c9c663",
  "metric": "DISPATCH_METRIC_AVERAGE_POSITION",
  "windowLength": "2",
  "transferInterval": "4",
  "entityScope": "ENTITY_SCOPE_TEAMS",
  "teamScope": [
    "0fc06d09f0805f1da96dd171af4a6e38dc6d443e135c855c7990a1ebe395b26f",
    "a33c89f67e0e8afda3165d34aeddba6082c674c3aa4ea17ece5213ef598c3558" ],
  "distributionStrategy": "DISTRIBUTION_STRATEGY_PRO_RATA"
  }
}
```
Obviously this wasn't broken in this PR, and I believe it's intentional. But... just in case

Reply:
@edd They're not full proposals, just a segment of a proposal. I don't think they should be valid in that submitting it in isolation will get you nowhere but if you think it's confusing, I can make them valid?

Reply:
I thought you'd probably say that. It's fine. I just got annoyed at the red marks in the diff view :sweat_smile: